docs/CHILI-GraFx/concepts/federated-single-sign-on/saml/index.md

```diff
@@ -1,24 +1,79 @@
-# Federated Single sign-on: SAML
+# Federated Single Sign On: SAML
 
-This document lists the info for your SAML Identity Providers in the CGX Platform.
+This document lists the steps required for setting up Federated Single Sign On from your SAML Identity Provider ("IDP") to CHILI GraFx.
 
-## CHILI GraFx will need
+## 1. Configure your IDP
 
-- Sign in URL 
-- X509 Signing Certificate 
-- Claim names for this information: 
-	- Email (default is email)
-	- First Name (default is given_name)
-	- Last Name (default is family_name)
-	- Group memberships (default is https://chili-publish.com/CGXGroups)
-- Identity attribute 
+Create a SAML application in your IDP.
 
-## We will provide you
+To do this, you will need a few details:
 
-- Authentication request signing certificate: Provided upon request. 
-- SP ID: Provided upon request. 
-- The identity attribute must be the user’s corporate email address 
+- Redirect URI: `https://login.chiligrafx.com/login/callback`
+- Service Provider ID
+  - The name GraFx will identify itself as on your IDP
+  - This will be provided by our Client Success team
+
+Please configure your IDP to provide at least following attributes in SAML responses:
```

Comment:
iirc, those are explicitely required, so it would be best to state them as such
`Please configure your IPD to provide required attributes in SAML responses:`

Reply:
added 'required'